app/views/viewlet-manager.js

```diff
@@ -266,13 +268,21 @@ YUI.add('juju-viewlet-manager', function(Y) {
         view.render(this.getAttrs());
         viewContainer.append(view.get('container'));
         this.databindingBind(model, view);
-      } else {
+      } else if (typeof view.render === 'function') {
```

Comment:
Yeah there are probably more robust techniques. This was just the easiest to get moving